Misc/NEWS.d/next/C API/2021-03-19-01-26-34.bpo-35134.TMWh1i.rst

```diff
@@ -0,0 +1,3 @@
+Move the non-limited C API of include/compile.h into the include/cpython
+directory. They must not be included directly, as they are already included by
+Python.h: :ref:`Include Files <api-includes>`.
```

Comment:
I don't think that it's worth it to mention it to users, since it's transparent for users.

Reply:
OK.